src/OrganisationRegistry.UI/app/organisation/info/cancel-coupling-with-kbo/cancel-coupling-with-kbo.template.html

```diff
@@ -0,0 +1,28 @@
+<div class="grid">
+    <div class="col--8-12 col--10-12--m col--1-1--s">
+      <div>
+        <h1 class="h2">Koppeling met de KBO ongedaan maken</h1>
+        <p>De velden 'Naam' en 'Korte naam' worden hiermee terug ingesteld op hun waarde voor de koppeling met de KBO.</p>
```

Comment:
And everything else from before coupling. Should we mention that?

Reply:
done.